ouroboros/main.py

```diff
@@ -10,10 +10,15 @@
 import image
 import cli
 from logger import set_logger
+from prometheus_client import start_http_server
+from prometheus_client import Counter
+
 
 def main():
     """Find running containers and update them with images using latest tag"""
     log = logging.getLogger(__name__)
+    # Setup our prometheus counter
+    updated_containers_counter = Counter('containers_updated', 'Count of containers updated')
```

Comment:
Name of counter might be more ideally `ouroboros_container_update_count`, based on the suggested naming convention: https://prometheus.io/docs/practices/naming/.